platform-sdk/docs/core/dynamic-address-book/DynamicAddressBook-PlatformDesign.md

```diff
@@ -0,0 +1,107 @@
+# Dynamic Address Book - Platform Design
+
+![Dynamic Address Book Platform Design](DynamicAddressBook-PlatformDesign.drawio.svg)
+
+## Network Connection Effects
+
+Assumption: That the network is fully connected, all nodes connect to each other.
+
+* Need to validate that this is true.
+* Huge gap in testing if not.
```

Comment:
I'm not sure this statement should be in the readme

Reply:
agreed

Reply:
I'm removing the assumption as well.  I'm leaving the question of partial networking connectivity to the Chatter effort.